x509/x509.go

```diff
@@ -1264,14 +1329,14 @@ func parsePublicKey(algo PublicKeyAlgorithm, keyData *publicKeyInfo, nfe *NonFat
 		if len(rest) != 0 {
 			return nil, errors.New("x509: trailing data after RSA public key")
 		}
-
```

Comment:
nit: you've removed a blank line here (sorry to be pedantic, but any unnecessary changes make upstream merges harder)